# generated by patch-package 6.4.8 on 2022-02-20 10:17:00
#
# command:
#   npx patch-package vscode-framework
#
# declared package:
#   vscode-framework: ^0.0.18
#
diff --git a/node_modules/vscode-framework/build/cli/configurationFromType.js b/node_modules/vscode-framework/build/cli/configurationFromType.js
index ea2d299..e89c194 100644
--- a/node_modules/vscode-framework/build/cli/configurationFromType.js
+++ b/node_modules/vscode-framework/build/cli/configurationFromType.js
@@ -72,6 +72,8 @@ const generateConfigFromFile = (configurationTypeFilePath) => {
         required: true,
         strictNullChecks: true,
         topRef: true,
+        validationKeywords: ['enumDescription', 'suggestSortText', 'defaultSnippets'],
+        ignoreErrors: true
     };
     console.time('TJS generate configuration');
     const program = TJS.getProgramFromFiles([configurationTypeFilePath], {});
